app/assets/stylesheets/main.scss

```diff
@@ -708,3 +709,21 @@ a {
 .text-complete{
   margin: 19px 0 0 0;
 }
+
+
+//RatherNotSay
+.rather-not-say-first-row {
+  margin-top: 240px;
+  div.row2 {
+    space: 'margin-top:-1rem';
```

Comment:
it is wrong style...  What I meant before is just change "margin-top:-1rem" to "margin-top: -1rem". Just add a simple space before -1rem. Update your code here to:

div.row2 {
    margin-top: -1rem;
    max-width: 600px;
    height: 75px;
    white-space: pre-wrap;
  }

Reply:
Last night I saw that this was not the case. Changed today. To be honest, I thought it was something new that I didn't know about